js/snake.js

```diff
@@ -57,7 +61,9 @@ class Game {
         var newHead = this.snake[0].slice();
         switch(this.nextMove) {
             case undefined:
-                return;
+                newHead[0] = newHead[0];
+                newHead[1] = newHead[1];
+                break;

```

Comment:
A simple return would be enough instead. If teh game is paused, you can return immediately from the `update` method